### PR TITLE
ConicIP - now with a tag

### PR DESCRIPTION
**Pull request by gabgoh**
sorry.

https://github.com/JuliaLang/METADATA.jl/pull/5837

### CODE REVIEW

**Comment by mlubin** [comment]
Hi @gabgoh. It's great to have this in Julia, feel free to announce on the julia-opt list.

Regarding the name, what do you think about ``ConicIPM``? (for interior-point method). [https://github.com/mlubin/Pajarito.jl](Pajarito) is a conic IP (integer programming) solver so I'm hoping we can avoid confusing potential users. Just expressing my preference here to keep the julia optimization ecosystem more coherent, but this isn't a veto.

CC @chriscoey

**Comment by mlubin** [comment]
CC @mpf

**Comment by timholy** [comment]
And to many people IP means Intellectual Property.

File: ConicIP/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4+
```
Comment (tkelman) [inline ConicIP/versions/0.1.0/requires]: this `+` means it won't be installable on any 0.4 release

**Comment by gabgoh** [comment]
I'm thinking ConicQP - QP stands for quadratic program. It has the same ring to IP but doesn't have the baggage of all the things it already means

**Comment by chriscoey** [comment]
But aren't you going to support SDP one day perhaps?

**Comment by mlubin** [comment]
SDP is already there from what I saw. QP sounds restrictive when the solver is in fact much more general.

**Comment by mlubin** [comment]
(Side note: What's the motivation for supporting convex quadratic objectives explicitly when they could just be handled via second-order cone constraints and an epigraph?)

**Comment by gabgoh** [comment]
ok back to the drawing board. QP does seem confer the notion of "polyhedral constraints".

@chriscoey SDPs are supported. 

@mlubin we had a brief debate about this, but it turns out that the extra freedom in having the Q explicitly as an option helps. For example, in my own work I use Q's with special structure (e.g. Q can be of type `Woodbury`) and the KKT systems are way nicer with Q explicit than implicit in an epigraphical SOC. Also, Q can be set to a sparse zero matrix if unused, so there's minimal overhead.

**Comment by mlubin** [comment]
@gabgoh, makes sense, thanks for the explanation!

**Comment by mpf** [comment]
This thread seems to have stalled. Is this an implicit veto for `ConicIP`? Otherwise, would be good proceed with registering the package with METADATA.

**Comment by tkelman** [comment]
If you feel strongly about keeping the name as-is, that's fine.

What is the intended minimum Julia version? Would at least like to get the require file correctly reflecting the supported versions.

**Comment by tkelman** [comment]
Looks like you changed it in the package repo, but that won't propagate to METADATA unless you either make another tag, or modify the existing files in this branch of your metadata fork.

**Comment by mlubin** [comment]
No veto here

**Comment by gabgoh** [comment]
ok closing this PR, creating a new one.

**Pull request closed without merging**